site2/website/pages/en/versions.js

```diff
@@ -1,13 +1,12 @@
+const React = require("react");
 
-const React = require('react');
-
-const CompLibrary = require('../../core/CompLibrary');
+const CompLibrary = require("../../core/CompLibrary");
```

Comment:
I just noticed that the file hasn't changed at all, right? It probably doesn't need to be formatted

Reply:
fixed, thanks.